go.mod

```diff
@@ -50,13 +50,13 @@ require (
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c
 	gopkg.in/tomb.v2 v2.0.0-20161208151619-d5d1b5820637
 
-	//pinned k8s.io to v0.23.5 tag
-	k8s.io/api v0.23.5
-	k8s.io/apiextensions-apiserver v0.23.5
-	k8s.io/apimachinery v0.23.5
-	k8s.io/cli-runtime v0.23.5
-	k8s.io/client-go v0.23.5
-	k8s.io/kubectl v0.23.5
+	//pinned k8s.io to v0.24.0 tag
```

Comment:
should the "patch" version be updated to the latest for all of these dependencies?

Reply:
we can, I will update it and see if there are any issues